rpm/SPECS/crowdsec.spec

```diff
@@ -39,6 +39,7 @@ BuildRequires:  systemd
 
 %build
 BUILD_VERSION=%{local_version} make
+sed -i "s#/usr/local/lib/crowdsec/plugins/#/usr/lib/crowdsec/plugins/#g" config/config.yaml
```

Comment:
Why sed and not a patch ?

Reply:
because it can be either `/usr/lib` or `/usr/lib64`. But still, this needs using macros...